pkg/node/address.go

```diff
@@ -7,6 +7,7 @@ import (
 	"bufio"
 	"errors"
 	"fmt"
+	ipamOption "github.com/cilium/cilium/pkg/ipam/option"
```

Comment:
you can run `goimports -w -local github.com/cilium/cilium address.go` which will reorder the imports to resolve this linting error

Reply:
ah indeed my bad, thanks! 🙇